### ISSUE TITLE
Add a 'pristine' property, or change the way 'dirty' behaves

### ISSUE THREAD

**Issue by cheapsteak** [kazupon/vue-validator#33]
I was hoping `dirty` would work the opposite of angular forms' `pristine`, but that doesn't seem to be exactly the case.  

It would be useful to have a property that keeps track of whether a model has ever been changed

Imagine a signup form with a required `username` field, initially empty.

I don't want to have an error showing up as soon as the page loads, telling the user they've done something wrong even though they haven't interacted with it yet.

I _do_ want to inform them that it's a required field if they type something in, then delete it

The way `dirty` currently behaves is that it only checks if current value != initial value, might be better if it was renamed to `modified` and have `dirty` keep track of whether the value was ever changed at all. Or have a `pristine` property that keeps track of the same thing.

As a reference, here's how [Angular's formcontroller](https://docs.angularjs.org/api/ng/type/form.FormController) handles the variables:

```
$pristine
boolean 
True if user has not interacted with the form yet.

$dirty
boolean 
True if user has already interacted with the form.
```

**Comment by kazupon** [comment]
Hmm..., you’re right.

In next version 2.0, I'll plan to full-scratch implementation (see #24).
I'll improve the this issue. :smiley:

**Comment by cheapsteak** [comment]
Thank you :)
Is there any way an interim property could be added to accommodate this use-case prior to v2's release?
Perhaps just adding the implementation for `pristine`?

**Comment by kazupon** [comment]
@cheapsteak 
Yes, is there.
I think so too.

**Comment by kazupon** [comment]
I published 2.0.0-alpha.1
